src/std/build.ss

```diff
@@ -13,4 +13,9 @@
 ;; The indirection below allows the build-std to do its imports in a module context, transitively
 (gx#import-module "build-std.ss" #t #t)
 ((eval 'std/build-std#build)
- srcdir: srcdir libdir: libdir debug: #t)
+ srcdir: srcdir libdir: libdir
+ debug:
+ (cond-expand
```

Comment:
this is not even here if you rebase snd drop the old commits from the bsd branch

Reply:
What do you mean? What branch? It's still there on master: https://github.com/mighty-gerbils/gerbil/blob/master/src/std/build.ss#L16

Reply:
Oh, wait, you mean I can re-add the debug for all platforms? I'll try that lol.